**Introduction Theory**

The iBright CL500 imaging system is an imaging device that allows users to image chemically illuminated Western blots, DNA and RNA gels with fluorescent nucleic acid stains, and visible protein gels. The device is run from a single screen and images can be further studied with its image analysis software.

The measurement functions the machine uses are to produce background corrected volume and density; as well as the weight and

migration distance of the gel. The distance of the gel ![](Aspose.Words.f2ee218a-db80-4138-9243-b5fcec2934fc.001.png)is based on the physics of DNA. Because the DNA has a negatively charged phosphate backbone, the DNA will migrate towards a positive anode and will be separated based on the speed that is inversely related to their lengths as seen in Figure 1. The imager uses this information when using its smart exposure feature to determine how big or small the DNA bands should appear, and during the calculation of the migration distance in its analysis report1. The main function of the iBright1500 is its image analysis that captures signals of various blots and gels using specific stains, substrates, and

fluorophores. For the nucleic acid gel specifically, the device can image ethidium bromide and Invitrogen SYBR stains1. For our use we stained the DNA with the Invitrogen NucBlue Reagent. The machine will shine a UV light on this stain to excite it at 360 nm and has a maximum emission at 460 nm, resulting in a large Stokes shift.

Equation 1: Stokes shift = Absorption(ùúÜ) - Emission(ùúÜ)

And the stain has a high affinity to DNA - making it useful for image analysis. After the gel stain is excited the imager proceeds to go through its smart exposure protocol to automatically determine the desired signal and limit overexposed or underexposed images. The camera is able to calculate its measurements because of this camera, and uses software to analyze the image. From the emission spectrum of the stain, the devices software can calculate a volume of the sample using the sum of pixel intensities contained in the bands; a weight estimation based on its relative mobility in the gel compared to known standards; a density value using the average intensity for all the pixels in a band; and a migration distance using the distance migrated divided by gel length1. These calculations utilize known concepts of DNA charge and speed, emission spectra physics, and software image analysis.

**Practice**

In practice, the iBright 1500 mainly functions through its touch screen at the top of the instrument. When imaging a sample with the iBright1500 it is important to select the right mode of configuration for the device. The options are all related to experiments available to run: chemiluminescent substrate imaging, DNA and RNA gels, stained protein gels, and Universal for multiple readouts. Afterwards, we have to place our sample in the drawer in the center of the viewing area. The machine will auto align, zoom, and focus the sample and will also acquire a live-view sample image so there is no need for physical interaction at this point.

Once the drawer is closed we are able to select smart exposure or fine tune the exposure time manually through the more options tab on the ![](Aspose.Words.f2ee218a-db80-4138-9243-b5fcec2934fc.002.png)screen seen in Figure (2). The smart exposure works by predicting the optimal exposure for minimizing pixel saturation and maximizing the dynamic range for a specific sample. It will also render a preview of how the image will appear at this exposure time1. Other uses of the more options tab include editing the signal accumulation where the device can set a series of exposures based on a first capture, last capture, and total number of captures; also editing saturation, inversion, and band excision (opens the drawer and activates a safe green transilluminator for Nucleic Acid gels mode). After the exposure time and other modes are set correctly we are able to take an image of the gel.

The image is analyzed through the iBright Analysis Software. After the image is taken, the interface for this software will appear as

a selection choice on the screen under ![](Aspose.Words.f2ee218a-db80-4138-9243-b5fcec2934fc.003.png)analysis. We can also export the image or view the gallery where it is automatically stored. In the software interface we can adjust the image by editing or removing lanes and bands, adding markers, normalizing the image (Figure 3) or generating a report1. The software will automatically find and display the analysis frames, lanes, and bands for the image. From this we can raise or lower the band-find threshold to increase or decrease the number of identified bands and lanes in the analysis. We can also select a dye from the drop-down menu to select a particular dye to view.

**Use Cases**

Our measurement technology is mainly used for thorough analysis of biological materials. Gel imaging often includes analysis software that can quantify the sample to determine band density, molecular weights and sizes, or other comparative analysis. One case study of a similar gel imaging technology to the iBright CL1500 is a study done on a kernel based approach to texture analysis of proteins. The study done by Fernandez-Lozano Et al. used ten 1024 x 1024 8 bit 2 DE images - similar to the imaging the iBright CL1500 does - in order to build a data set to test ML classification systems on. The images were visualized using silver staining standard protocols. In order to test the effectiveness of their model they used texture analysis based on patterns of homogeneity amongst spatially close pixels and ran this data through various LLMs2. A second study that used gel imaging was done by Sven Luhn et. al on organizing spot data based on the concept of a standard position for a protein species - similar to the iBright device‚Äôs protein gel image mode. In order to do so, their method was to use 23 gel images of bacteria Bacillus subtilis. The images were abstained by radiolabeling crude protein extracts from *B. subtilis* cultures. The proteins were exposed to storage phosphor screens and then scanned using a PhosphorImager SI. And the resolution was 100 um and had a 16-bit color depth3. This was done to speculate that protein species would accumulate in only one spot. While the images correctly identify most of the protein species, some of them appear to be blurry as they are close to others. A third experiment was aimed to implement and evaluate an image analysis protocol with an open-source software that can identify protein clusters in 2D gel electrophoresis images. Molina-Mora Et al. studied periplasmic proteins of *Pseudomonas aeruginosa* under various conditions of antibiotic exposure. In order to capture and process the images, the group used a silver staining protocol similar to the first experiment previously discussed. The images were then auto aligned similar to the iBright software; however, this experiment used software that could align all of the photos they took to a single reference. Then, a spot identification protocol was used to identify the total number and cluster grouping of the protein spots4. This experiment utilized a similar software alignment and gel analysis to the iBright device. But, this study went further than what the iBright software could do and cross referenced alignment angles of the images in order to better the inconsistency seen in 2D gel electrophoresis image analysis experiments.

Based on these experiments showcasing the use of gel imaging in the science field, an experiment we can run on the instrument in the lab could be used to locate clusters of variations of tau proteins found in CTE and Alzheimers affected individuals. Many of the studies publications remarked on the lack of some, ‚Äúmass spectrometer technology to be able to routinely distinguish all modifications of a protein that yield distinct spots‚Äù 3. In order to further the research into tau protein identification and coordination across the field our experiment would focus on the genetic variation of tau protein construction and identification. After properly setting up the protein variants we would use the iBight software in order to study the classification of these proteins and the applications into their detectability in blood samples. The iBright machine would allow us to identify new variations of tau proteins through GE protocols and further research into the consistency in detection of these proteins not yet properly developed in 2D-GE protocol.

**Materials and Methods**

In this experiment, the iBright 1500 Gel imager was used to fluorescently visualize the dilution of a DNA ladder. Protocol was followed from BTEC iBright Demo5, with a few minor modifications to adjust for what materials were available. First, an agarose gel solution was prepared in a 200mL erlenmeyer flask using 1g Invitrogen UltraPure agarose with 100mL 1xTAE buffer (diluted from 10x solution). The solution was microwaved until it appeared uniform and clear, about 1 minute. Solution was allowed to cool for about 5 minutes, then was poured into a gel tray with a pre-inserted well comb, contained in an electrophoresis voltage chamber (Thermo Scientific Owl EasyCast B1-BP). The gel was allowed to sit in the fridge until completely solidified, about 20-30 minutes. While the gel was setting, DNA ladder dilutions were prepared according to the following concentrations:

Table 1: Concentrations of DNA ladder/loading dye, water, and fluorescent beads in each dilution of the ladder

||1x Dilution|2x Dilution|6x dilution|
| :- | - | - | - |
|DNA (uL)|1|3|6|
|Water (uL)|5|3|0|
|Fluorescence (uL)|6|6|6|

The materials used to make the three samples were the Invitrogen 1 Kb DNA ladder with pre-mixed loading dye, DI water, and Invitrogen NucBlue Ready Probes (fluorescence).

Once the gel was fully solidified, well comb was carefully removed and the agarose gel was placed back into electrophoresis box such that the wells created by the comb were on the side of the negative electrode port, and the opposite end of the gel was on the side of the positive electrode port. 1xTAE buffer was poured into the box until it just covered the top of the gel. The whole 12uL of each of the three DNA samples were loaded into their own well. Then, the power supply (Thermo Scientific EC 3000 XL) was connected and set to 150V, and the gel ran for approximately 30 minutes until the bands could be seen around halfway down the gel (if more time was available, it would have been preferable to run the gel until the bands were 75-80% down the gel).

The power supply was turned off and the gel was removed from the electrophoresis box and transferred to the gel compartment of the iBright 1500 imager. The setting used was Nucleic Acid Gels, and the image of the gel was fluorescently captured using Smart Exposure. The image itself, as well as the report the iBright generated, were saved and exported to a USB.

Finally, the information produced by the iBright was used in t-tests to determine the significance of the effects of DNA concentration on intensity, area, and migration distance of the DNA bands.

**Results**

The final results of the gel electrophoresis are demonstrated in Figures 4 and 5. Figure 4 depicts the results of the gel electrophoresis as seen by the naked eye and Figure 4 depicts the results of the gel electrophoresis captured by the iBright imager. Bands of the DNA ladder can be seen as blue and yellow bands under the [2x] and [6x] columns in Figure 4, but the [1x] column was too dilute to be seen to the naked eye. The iBright gel imager allows us to view the complete DNA ladder via fluorescent imaging (Figure 5).

` `![](Aspose.Words.f2ee218a-db80-4138-9243-b5fcec2934fc.004.png)![](Aspose.Words.f2ee218a-db80-4138-9243-b5fcec2934fc.005.png)

The iBright Imager provides a detailed analysis of each band of the DNA ladder as seen in Figure 5. The device provides a full report of the nucleic acid analysis that includes the median fluorescence intensity, area in pixels, and Rf value (migration distance) for each band identified

in Figure 6. It is important to note that the gel imager did not capture all the bands present in the![](Aspose.Words.f2ee218a-db80-4138-9243-b5fcec2934fc.006.png)

[6x] concentrated DNA in the other lanes. This can be adjusted on the gel imager machine by lowering the threshold intensity value or increasing the exposure time.

As expected the higher concentrations of the DNA ladder have higher intensities (Figure 7). This is

also supported by our statistical analysis. T-tests were performed to compare the average intensity, area in pixels, and Rf value of the bands between the three concentrations. The average intensity of the gel bands increases with concentration (p-value < 0.01). Figures 7 and 8 further illustrate these findings. The average area and the Rf value does not change with concentration (p-value = 0.55, p-value = 0.16). In figure 7 it is important to note that the gel imager did not identify bands 3, 4, 5, and 6, in the [1x] DNA concentration and that is why their intensities are plotted as zero in the figure.

![](Aspose.Words.f2ee218a-db80-4138-9243-b5fcec2934fc.007.png)![](Aspose.Words.f2ee218a-db80-4138-9243-b5fcec2934fc.008.png)

**Discussion**

This relatively simple DNA dilution experiment unveiled some interesting dynamics underlying gel electrophoresis. Despite the expected linear relationship between magnitude of dilution and fluorescent intensity, non-linear dynamics were observed likely due to the saturation of the imager at the set exposure time and the saturation of the fluorescent dye. At the set exposure time, as the DNA ladders become more concentrated, the fluorescent intensity leaves the dynamic range of the camera, leading to a saturation regime in intensity. In addition, consumption of the fluorescent dye becomes consequential, as DNA becomes the limiting reagent rather than the dye. Both saturation effects are evident in Figure 8 where the median intensity of each band starts to reach a constant as DNA concentration increases. In terms of statistical analysis, two-sided t-tests revealed expected results: that fluorescence intensity significantly varies with DNA concentration while the resultant area of DNA bands do not. Moreover, these tests demonstrate there is a decent variation in the migration distances even though it does not reach the level of significance across concentrations, as one would expect of the DNA samples with the same distribution of sizes. This highlights the limitations of gel electrophoresis and gel imaging which is that it is prone to human error and the stochasticity

one would expect from a potential driven random walk6. It also has low throughput, as the gel had only 8 channels, suffers from low resolution, where the gaussian peaks of bands with similar molecular weights cannot be resolved, cannot detect low concentrations of protein, relies of the specificity of probes, which for proteins tends to lead to off target effects, and only tells you what is present in a homogenized sample7.

To combat the low throughput and the lack of variance data between samples (i.e between tissues and cells with respect to DNA, RNA, and proteins), one could simply employ a microfluidic approach: lower the scales to get more samples per same area of gel with well-behaved flows. By effectively making each well much smaller, thereby requiring some automated process to fill and track reagents as well as a more sensitive imager, one can increase the throughput and analyze magnitudes more samples, potentially useful for large scale mutational studies or drug target screening. In addition, by choosing each well to be an independent single cell lysate rather than the lysate of a homogenized sample, one can better capture the heterogeneity in nucleic acid or protein composition. Moreover, with proper stripping and reprobing protocols, one can multiplex probes to further analyze protein composition, addressing another traditional limitation7.

![](Aspose.Words.f2ee218a-db80-4138-9243-b5fcec2934fc.009.png)

With all these considerations in retrospect, the experiment is fairly well-suited to the device. Despite its simple design, it allowed us to unveil some surprising dynamics of electrophoresis and the imager by simply changing the DNA ladder concentrations. Due to its simple nature, it was well within the range of the devices capabilities (i.e low throughput, known specific probes, high DNA concentration). However, it was largely an experiment for the device, as opposed to an experiment that leverages the device and to improve the next iteration of experiments, it would be interesting to use it in a more applied context, such as conducting a western blot for HIV viral proteins9.

**References**

1. ThermoFisher Scientific. (2019). iBright Imaging Systems User Guide. In User guide. ThermoFisher Scientific.
1. Fernandez-Lozano, C., Seoane, J., Gestal, M. et al. Texture analysis in gel electrophoresis images using an integrative kernel-based approach. Sci Rep 6, 19256 (2016). <https://doi.org/10.1038/srep19256>
1. Luhn S, Berth M, Hecker M, Bernhardt J. Using standard positions and image fusion to create proteome maps from collections of two-dimensional gel electrophoresis images. Proteomics. 2003 Jul;3(7):1117-27. doi: 10.1002/pmic.200300433. PMID: 12872213.
1. Molina-Mora JA, Chinchilla-Montero D, Castro-Pe√±a C, Garc√≠a F. Two-dimensional gel electrophoresis (2D-GE) image analysis based on CellProfiler: Pseudomonas aeruginosa AG1 as model. Medicine (Baltimore). 2020 Dec 4;99(49):e23373. doi: 10.1097/MD.0000000000023373. PMID: 33285719; PMCID: PMC7717798.
1. BTEC iBright Demo Protocol
1. Gilda JE, Ghosh R, Cheah JX, West TM, Bodine SC, Gomes AV. Western Blotting

   Inaccuracies with Unverified Antibodies: Need for a Western Blotting Minimal Reporting Standard (WBMRS). PLoS One. 2015 Aug 19;10(8):e0135392. doi: 10.1371/journal.pone.0135392. PMID: 26287535; PMCID: PMC4545415.

7. Ziraldo R, Shoura MJ, Fire AZ, Levene SD. Deconvolution of nucleic-acid length distributions: a gel electrophoresis analysis tool and applications. Nucleic Acids Res. 2019 Sep 19;47(16):e92. doi: 10.1093/nar/gkz534. PMID: 31226202; PMCID: PMC6895257.
7. Mishra M, Tiwari S, Gomes AV. Protein purification and analysis: next generation Western blotting techniques. Expert Rev Proteomics. 2017 Nov;14(11):1037-1053. doi: 10.1080/14789450.2017.1388167. Epub 2017 Oct 13. PMID: 28974114; PMCID: PMC6810642.
7. Meles H, Wolday D, Fontanet A, Tsegaye A, Tilahun T, Aklilu M, Sanders E, De Wit TF. Indeterminate human immunodeficiency virus Western blot profiles in ethiopians with discordant screening-assay results. Clin Diagn Lab Immunol. 2002 Jan;9(1):160-3. doi: 10.1128/cdli.9.1.160-163.2002. Erratum in: Clin Diagn Lab Immunol. 2008 Aug;29(8):1215. PMID: 11777847; PMCID: PMC119890.
